### DISCUSSION TITLE
Aggiornamento 2023.6.3 errore ha-av

### DISCUSSION THREAD

**Discussion by Ilmessaggeromascherato** [indomus/forum#2524] [HUB: Home Assistant]
Buongiorno 
ho fatto l'aggiornamento a 2023.6.3 ma mi è uscito l'errore su stream - deafult relativo a ah-av 10.1.0. 
Anche a mano non riesco ad installarlo. 
Non viglio discutere se meglio Venv o docker. In questo momento chiedo se qualcuno di voi sa come risolvere il problema. How seguito le guide trovate sui vari post, ma senza successo. 
Qualcuno sa darmi una diritta ? 
Grazie

**Comment by snakuzzo** [comment]
Per cortesia posta l'errore "preciso" che vedi nei log di Core e cosa fai per installare quello che manca

**Reply by Ilmessaggeromascherato** [reply]
Ciao scusami 
lo copio qui sotto spero vada bene. 
Grazie 

`
Logger: homeassistant.util.package Source: util/package.py:107  First occurred: 2 luglio 2023, 17:51:12 (3 occurrences)  Last logged: 2 luglio 2023, 17:57:51
Unable to install package ha-av==10.1.0: error: subprocess-exited-with-error × Building wheel for ha-av (pyproject.toml) did not run successfully. │ exit code: 1 ╰─> [172 lines of output] running bdist_wheel running build running build_py creating build creating build/lib.linux-aarch64-cpython-311 creating build/lib.linux-aarch64-cpython-311/av copying av/__init__.py -> build/lib.linux-aarch64-cpython-311/av copying av/about.py -> build/lib.linux-aarch64-cpython-311/av copying av/deprecation.py -> build/lib.linux-aarch64-cpython-311/av copying av/__main__.py -> build/lib.linux-aarch64-cpython-311/av copying av/datasets.py -> build/lib.linux-aarch64-cpython-311/av creating build/lib.linux-aarch64-cpython-311/av/audio copying av/audio/__init__.py -> build/lib.linux-aarch64-cpython-311/av/audio creating build/lib.linux-aarch64-cpython-311/av/sidedata copying av/sidedata/__init__.py -> build/lib.linux-aarch64-cpython-311/av/sidedata creating build/lib.linux-aarch64-cpython-311/av/codec copying av/codec/__init__.py -> build/lib.linux-aarch64-cpython-311/av/codec creating build/lib.linux-aarch64-cpython-311/av/container copying av/container/__init__.py -> build/lib.linux-aarch64-cpython-311/av/container creating build/lib.linux-aarch64-cpython-311/av/video copying av/video/__init__.py -> build/lib.linux-aarch64-cpython-311/av/video creating build/lib.linux-aarch64-cpython-311/av/bitstream copying av/bitstream/__init__.py -> build/lib.linux-aarch64-cpython-311/av/bitstream creating build/lib.linux-aarch64-cpython-311/av/filter copying av/filter/__init__.py -> build/lib.linux-aarch64-cpython-311/av/filter creating build/lib.linux-aarch64-cpython-311/av/subtitles copying av/subtitles/__init__.py -> build/lib.linux-aarch64-cpython-311/av/subtitles creating build/lib.linux-aarch64-cpython-311/av/data copying av/data/__init__.py -> build/lib.linux-aarch64-cpython-311/av/data copying av/buffer.pxd -> build/lib.linux-aarch64-cpython-311/av copying av/stream.pxd -> build/lib.linux-aarch64-cpython-311/av copying av/packet.pxd -> build/lib.linux-aarch64-cpython-311/av copying av/dictionary.pxd -> build/lib.linux-aarch64-cpython-311/av copying av/logging.pxd -> build/lib.linux-aarch64-cpython-311/av copying av/format.pxd -> build/lib.linux-aarch64-cpython-311/av copying av/error.pxd -> build/lib.linux-aarch64-cpython-311/av copying av/plane.pxd -> build/lib.linux-aarch64-cpython-311/av copying av/utils.pxd -> build/lib.linux-aarch64-cpython-311/av copying av/bytesource.pxd -> build/lib.linux-aarch64-cpython-311/av copying av/descriptor.pxd -> build/lib.linux-aarch64-cpython-311/av copying av/frame.pxd -> build/lib.linux-aarch64-cpython-311/av copying av/__init__.pxd -> build/lib.linux-aarch64-cpython-311/av copying av/enum.pxd -> build/lib.linux-aarch64-cpython-311/av copying av/option.pxd -> build/lib.linux-aarch64-cpython-311/av copying av/audio/resampler.pxd -> build/lib.linux-aarch64-cpython-311/av/audio copying av/audio/stream.pxd -> build/lib.linux-aarch64-cpython-311/av/audio copying av/audio/codeccontext.pxd -> build/lib.linux-aarch64-cpython-311/av/audio copying av/audio/format.pxd -> build/lib.linux-aarch64-cpython-311/av/audio copying av/audio/layout.pxd -> build/lib.linux-aarch64-cpython-311/av/audio copying av/audio/plane.pxd -> build/lib.linux-aarch64-cpython-311/av/audio copying av/audio/frame.pxd -> build/lib.linux-aarch64-cpython-311/av/audio copying av/audio/__init__.pxd -> build/lib.linux-aarch64-cpython-311/av/audio copying av/audio/fifo.pxd -> build/lib.linux-aarch64-cpython-311/av/audio copying av/sidedata/sidedata.pxd -> build/lib.linux-aarch64-cpython-311/av/sidedata copying av/sidedata/__init__.pxd -> build/lib.linux-aarch64-cpython-311/av/sidedata copying av/sidedata/motionvectors.pxd -> build/lib.linux-aarch64-cpython-311/av/sidedata copying av/codec/codec.pxd -> build/lib.linux-aarch64-cpython-311/av/codec copying av/codec/__init__.pxd -> build/lib.linux-aarch64-cpython-311/av/codec copying av/codec/context.pxd -> build/lib.linux-aarch64-cpython-311/av/codec copying av/container/core.pxd -> build/lib.linux-aarch64-cpython-311/av/container copying av/container/streams.pxd -> build/lib.linux-aarch64-cpython-311/av/container copying av/container/output.pxd -> build/lib.linux-aarch64-cpython-311/av/container copying av/container/pyio.pxd -> build/lib.linux-aarch64-cpython-311/av/container copying av/container/__init__.pxd -> build/lib.linux-aarch64-cpython-311/av/container copying av/container/input.pxd -> build/lib.linux-aarch64-cpython-311/av/container copying av/video/stream.pxd -> build/lib.linux-aarch64-cpython-311/av/video copying av/video/codeccontext.pxd -> build/lib.linux-aarch64-cpython-311/av/video copying av/video/format.pxd -> build/lib.linux-aarch64-cpython-311/av/video copying av/video/reformatter.pxd -> build/lib.linux-aarch64-cpython-311/av/video copying av/video/plane.pxd -> build/lib.linux-aarch64-cpython-311/av/video copying av/video/frame.pxd -> build/lib.linux-aarch64-cpython-311/av/video copying av/video/__init__.pxd -> build/lib.linux-aarch64-cpython-311/av/video copying av/bitstream/filter.pxd -> build/lib.linux-aarch64-cpython-311/av/bitstream copying av/bitstream/context.pxd -> build/lib.linux-aarch64-cpython-311/av/bitstream copying av/filter/graph.pxd -> build/lib.linux-aarch64-cpython-311/av/filter copying av/filter/pad.pxd -> build/lib.linux-aarch64-cpython-311/av/filter copying av/filter/filter.pxd -> build/lib.linux-aarch64-cpython-311/av/filter copying av/filter/link.pxd -> build/lib.linux-aarch64-cpython-311/av/filter copying av/filter/__init__.pxd -> build/lib.linux-aarch64-cpython-311/av/filter copying av/filter/context.pxd -> build/lib.linux-aarch64-cpython-311/av/filter copying av/subtitles/subtitle.pxd -> build/lib.linux-aarch64-cpython-311/av/subtitles copying av/subtitles/stream.pxd -> build/lib.linux-aarch64-cpython-311/av/subtitles copying av/subtitles/codeccontext.pxd -> build/lib.linux-aarch64-cpython-311/av/subtitles copying av/subtitles/__init__.pxd -> build/lib.linux-aarch64-cpython-311/av/subtitles copying av/data/stream.pxd -> build/lib.linux-aarch64-cpython-311/av/data copying av/data/__init__.pxd -> build/lib.linux-aarch64-cpython-311/av/data running build_ext building 'av._core' extension creating build/temp.linux-aarch64-cpython-311 creating build/temp.linux-aarch64-cpython-311/src creating build/temp.linux-aarch64-cpython-311/src/av gcc -pthread -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -fPIC -I/usr/include/arm-linux-gnueabihf -I/srv/homeassistant/include -I/usr/local/include/python3.11 -c src/av/_core.c -o build/temp.linux-aarch64-cpython-311/src/av/_core.o gcc -pthread -shared build/temp.linux-aarch64-cpython-311/src/av/_core.o -lavformat -lavcodec -lavdevice -lavutil -lavfilter -lswscale -lswresample -o build/lib.linux-aarch64-cpython-311/av/_core.cpython-311-arm-linux-gnueabihf.so building 'av.logging' extension gcc -pthread -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -fPIC -I/usr/include/arm-linux-gnueabihf -I/srv/homeassistant/include -I/usr/local/include/python3.11 -c src/av/logging.c -o build/temp.linux-aarch64-cpython-311/src/av/logging.o gcc -pthread -shared build/temp.linux-aarch64-cpython-311/src/av/logging.o -lavformat -lavcodec -lavdevice -lavutil -lavfilter -lswscale -lswresample -o build/lib.linux-aarch64-cpython-311/av/logging.cpython-311-arm-linux-gnueabihf.so building 'av.descriptor' extension gcc -pthread -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -fPIC -I/usr/include/arm-linux-gnueabihf -I/srv/homeassistant/include -I/usr/local/include/python3.11 -c src/av/descriptor.c -o build/temp.linux-aarch64-cpython-311/src/av/descriptor.o gcc -pthread -shared build/temp.linux-aarch64-cpython-311/src/av/descriptor.o -lavformat -lavcodec -lavdevice -lavutil -lavfilter -lswscale -lswresample -o build/lib.linux-aarch64-cpython-311/av/descriptor.cpython-311-arm-linux-gnueabihf.so building 'av.frame' extension gcc -pthread -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -fPIC -I/usr/include/arm-linux-gnueabihf -I/srv/homeassistant/include -I/usr/local/include/python3.11 -c src/av/frame.c -o build/temp.linux-aarch64-cpython-311/src/av/frame.o gcc -pthread -shared build/temp.linux-aarch64-cpython-311/src/av/frame.o -lavformat -lavcodec -lavdevice -lavutil -lavfilter -lswscale -lswresample -o build/lib.linux-aarch64-cpython-311/av/frame.cpython-311-arm-linux-gnueabihf.so building 'av.error' extension gcc -pthread -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -fPIC -I/usr/include/arm-linux-gnueabihf -I/srv/homeassistant/include -I/usr/local/include/python3.11 -c src/av/error.c -o build/temp.linux-aarch64-cpython-311/src/av/error.o gcc -pthread -shared build/temp.linux-aarch64-cpython-311/src/av/error.o -lavformat -lavcodec -lavdevice -lavutil -lavfilter -lswscale -lswresample -o build/lib.linux-aarch64-cpython-311/av/error.cpython-311-arm-linux-gnueabihf.so building 'av.format' extension gcc -pthread -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -fPIC -I/usr/include/arm-linux-gnueabihf -I/srv/homeassistant/include -I/usr/local/include/python3.11 -c src/av/format.c -o build/temp.linux-aarch64-cpython-311/src/av/format.o gcc -pthread -shared build/temp.linux-aarch64-cpython-311/src/av/format.o -lavformat -lavcodec -lavdevice -lavutil -lavfilter -lswscale -lswresample -o build/lib.linux-aarch64-cpython-311/av/format.cpython-311-arm-linux-gnueabihf.so building 'av.stream' extension gcc -pthread -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -fPIC -I/usr/include/arm-linux-gnueabihf -I/srv/homeassistant/include -I/usr/local/include/python3.11 -c src/av/stream.c -o build/temp.linux-aarch64-cpython-311/src/av/stream.o gcc -pthread -shared build/temp.linux-aarch64-cpython-311/src/av/stream.o -lavformat -lavcodec -lavdevice -lavutil -lavfilter -lswscale -lswresample -o build/lib.linux-aarch64-cpython-311/av/stream.cpython-311-arm-linux-gnueabihf.so building 'av.bytesource' extension gcc -pthread -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -fPIC -I/usr/include/arm-linux-gnueabihf -I/srv/homeassistant/include -I/usr/local/include/python3.11 -c src/av/bytesource.c -o build/temp.linux-aarch64-cpython-311/src/av/bytesource.o gcc -pthread -shared build/temp.linux-aarch64-cpython-311/src/av/bytesource.o -lavformat -lavcodec -lavdevice -lavutil -lavfilter -lswscale -lswresample -o build/lib.linux-aarch64-cpython-311/av/bytesource.cpython-311-arm-linux-gnueabihf.so building 'av.buffer' extension gcc -pthread -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -fPIC -I/usr/include/arm-linux-gnueabihf -I/srv/homeassistant/include -I/usr/local/include/python3.11 -c src/av/buffer.c -o build/temp.linux-aarch64-cpython-311/src/av/buffer.o gcc -pthread -shared build/temp.linux-aarch64-cpython-311/src/av/buffer.o -lavformat -lavcodec -lavdevice -lavutil -lavfilter -lswscale -lswresample -o build/lib.linux-aarch64-cpython-311/av/buffer.cpython-311-arm-linux-gnueabihf.so building 'av.enum' extension gcc -pthread -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -fPIC -I/usr/include/arm-linux-gnueabihf -I/srv/homeassistant/include -I/usr/local/include/python3.11 -c src/av/enum.c -o build/temp.linux-aarch64-cpython-311/src/av/enum.o gcc -pthread -shared build/temp.linux-aarch64-cpython-311/src/av/enum.o -lavformat -lavcodec -lavdevice -lavutil -lavfilter -lswscale -lswresample -o build/lib.linux-aarch64-cpython-311/av/enum.cpython-311-arm-linux-gnueabihf.so building 'av.packet' extension gcc -pthread -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -fPIC -I/usr/include/arm-linux-gnueabihf -I/srv/homeassistant/include -I/usr/local/include/python3.11 -c src/av/packet.c -o build/temp.linux-aarch64-cpython-311/src/av/packet.o gcc -pthread -shared build/temp.linux-aarch64-cpython-311/src/av/packet.o -lavformat -lavcodec -lavdevice -lavutil -lavfilter -lswscale -lswresample -o build/lib.linux-aarch64-cpython-311/av/packet.cpython-311-arm-linux-gnueabihf.so building 'av.utils' extension gcc -pthread -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -fPIC -I/usr/include/arm-linux-gnueabihf -I/srv/homeassistant/include -I/usr/local/include/python3.11 -c src/av/utils.c -o build/temp.linux-aarch64-cpython-311/src/av/utils.o gcc -pthread -shared build/temp.linux-aarch64-cpython-311/src/av/utils.o -lavformat -lavcodec -lavdevice -lavutil -lavfilter -lswscale -lswresample -o build/lib.linux-aarch64-cpython-311/av/utils.cpython-311-arm-linux-gnueabihf.so building 'av.option' extension gcc -pthread -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -fPIC -I/usr/include/arm-linux-gnueabihf -I/srv/homeassistant/include -I/usr/local/include/python3.11 -c src/av/option.c -o build/temp.linux-aarch64-cpython-311/src/av/option.o gcc -pthread -shared build/temp.linux-aarch64-cpython-311/src/av/option.o -lavformat -lavcodec -lavdevice -lavutil -lavfilter -lswscale -lswresample -o build/lib.linux-aarch64-cpython-311/av/option.cpython-311-arm-linux-gnueabihf.so building 'av.plane' extension gcc -pthread -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -fPIC -I/usr/include/arm-linux-gnueabihf -I/srv/homeassistant/include -I/usr/local/include/python3.11 -c src/av/plane.c -o build/temp.linux-aarch64-cpython-311/src/av/plane.o gcc -pthread -shared build/temp.linux-aarch64-cpython-311/src/av/plane.o -lavformat -lavcodec -lavdevice -lavutil -lavfilter -lswscale -lswresample -o build/lib.linux-aarch64-cpython-311/av/plane.cpython-311-arm-linux-gnueabihf.so building 'av.dictionary' extension gcc -pthread -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -fPIC -I/usr/include/arm-linux-gnueabihf -I/srv/homeassistant/include -I/usr/local/include/python3.11 -c src/av/dictionary.c -o build/temp.linux-aarch64-cpython-311/src/av/dictionary.o gcc -pthread -shared build/temp.linux-aarch64-cpython-311/src/av/dictionary.o -lavformat -lavcodec -lavdevice -lavutil -lavfilter -lswscale -lswresample -o build/lib.linux-aarch64-cpython-311/av/dictionary.cpython-311-arm-linux-gnueabihf.so building 'av.audio.resampler' extension creating build/temp.linux-aarch64-cpython-311/src/av/audio gcc -pthread -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -fPIC -I/usr/include/arm-linux-gnueabihf -I/srv/homeassistant/include -I/usr/local/include/python3.11 -c src/av/audio/resampler.c -o build/temp.linux-aarch64-cpython-311/src/av/audio/resampler.o gcc -pthread -shared build/temp.linux-aarch64-cpython-311/src/av/audio/resampler.o -lavformat -lavcodec -lavdevice -lavutil -lavfilter -lswscale -lswresample -o build/lib.linux-aarch64-cpython-311/av/audio/resampler.cpython-311-arm-linux-gnueabihf.so building 'av.audio.frame' extension gcc -pthread -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -fPIC -I/usr/include/arm-linux-gnueabihf -I/srv/homeassistant/include -I/usr/local/include/python3.11 -c src/av/audio/frame.c -o build/temp.linux-aarch64-cpython-311/src/av/audio/frame.o gcc -pthread -shared build/temp.linux-aarch64-cpython-311/src/av/audio/frame.o -lavformat -lavcodec -lavdevice -lavutil -lavfilter -lswscale -lswresample -o build/lib.linux-aarch64-cpython-311/av/audio/frame.cpython-311-arm-linux-gnueabihf.so building 'av.audio.layout' extension gcc -pthread -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -fPIC -I/usr/include/arm-linux-gnueabihf -I/srv/homeassistant/include -I/usr/local/include/python3.11 -c src/av/audio/layout.c -o build/temp.linux-aarch64-cpython-311/src/av/audio/layout.o gcc -pthread -shared build/temp.linux-aarch64-cpython-311/src/av/audio/layout.o -lavformat -lavcodec -lavdevice -lavutil -lavfilter -lswscale -lswresample -o build/lib.linux-aarch64-cpython-311/av/audio/layout.cpython-311-arm-linux-gnueabihf.so building 'av.audio.format' extension gcc -pthread -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -fPIC -I/usr/include/arm-linux-gnueabihf -I/srv/homeassistant/include -I/usr/local/include/python3.11 -c src/av/audio/format.c -o build/temp.linux-aarch64-cpython-311/src/av/audio/format.o gcc -pthread -shared build/temp.linux-aarch64-cpython-311/src/av/audio/format.o -lavformat -lavcodec -lavdevice -lavutil -lavfilter -lswscale -lswresample -o build/lib.linux-aarch64-cpython-311/av/audio/format.cpython-311-arm-linux-gnueabihf.so building 'av.audio.codeccontext' extension gcc -pthread -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -fPIC -I/usr/include/arm-linux-gnueabihf -I/srv/homeassistant/include -I/usr/local/include/python3.11 -c src/av/audio/codeccontext.c -o build/temp.linux-aarch64-cpython-311/src/av/audio/codeccontext.o gcc -pthread -shared build/temp.linux-aarch64-cpython-311/src/av/audio/codeccontext.o -lavformat -lavcodec -lavdevice -lavutil -lavfilter -lswscale -lswresample -o build/lib.linux-aarch64-cpython-311/av/audio/codeccontext.cpython-311-arm-linux-gnueabihf.so building 'av.audio.stream' extension gcc -pthread -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -fPIC -I/usr/include/arm-linux-gnueabihf -I/srv/homeassistant/include -I/usr/local/include/python3.11 -c src/av/audio/stream.c -o build/temp.linux-aarch64-cpython-311/src/av/audio/stream.o gcc -pthread -shared build/temp.linux-aarch64-cpython-311/src/av/audio/stream.o -lavformat -lavcodec -lavdevice -lavutil -lavfilter -lswscale -lswresample -o build/lib.linux-aarch64-cpython-311/av/audio/stream.cpython-311-arm-linux-gnueabihf.so building 'av.audio.plane' extension gcc -pthread -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -fPIC -I/usr/include/arm-linux-gnueabihf -I/srv/homeassistant/include -I/usr/local/include/python3.11 -c src/av/audio/plane.c -o build/temp.linux-aarch64-cpython-311/src/av/audio/plane.o gcc -pthread -shared build/temp.linux-aarch64-cpython-311/src/av/audio/plane.o -lavformat -lavcodec -lavdevice -lavutil -lavfilter -lswscale -lswresample -o build/lib.linux-aarch64-cpython-311/av/audio/plane.cpython-311-arm-linux-gnueabihf.so building 'av.audio.fifo' extension gcc -pthread -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -fPIC -I/usr/include/arm-linux-gnueabihf -I/srv/homeassistant/include -I/usr/local/include/python3.11 -c src/av/audio/fifo.c -o build/temp.linux-aarch64-cpython-311/src/av/audio/fifo.o gcc -pthread -shared build/temp.linux-aarch64-cpython-311/src/av/audio/fifo.o -lavformat -lavcodec -lavdevice -lavutil -lavfilter -lswscale -lswresample -o build/lib.linux-aarch64-cpython-311/av/audio/fifo.cpython-311-arm-linux-gnueabihf.so building 'av.sidedata.motionvectors' extension creating build/temp.linux-aarch64-cpython-311/src/av/sidedata gcc -pthread -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -fPIC -I/usr/include/arm-linux-gnueabihf -I/srv/homeassistant/include -I/usr/local/include/python3.11 -c src/av/sidedata/motionvectors.c -o build/temp.linux-aarch64-cpython-311/src/av/sidedata/motionvectors.o gcc -pthread -shared build/temp.linux-aarch64-cpython-311/src/av/sidedata/motionvectors.o -lavformat -lavcodec -lavdevice -lavutil -lavfilter -lswscale -lswresample -o build/lib.linux-aarch64-cpython-311/av/sidedata/motionvectors.cpython-311-arm-linux-gnueabihf.so building 'av.sidedata.sidedata' extension gcc -pthread -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -fPIC -I/usr/include/arm-linux-gnueabihf -I/srv/homeassistant/include -I/usr/local/include/python3.11 -c src/av/sidedata/sidedata.c -o build/temp.linux-aarch64-cpython-311/src/av/sidedata/sidedata.o src/av/sidedata/sidedata.c: In function ‘__pyx_pymod_exec_sidedata’: src/av/sidedata/sidedata.c:5662:9: error: ‘AV_FRAME_DATA_SEI_UNREGISTERED’ undeclared (first use in this function) 5662 | if (((AV_FRAME_DATA_SEI_UNREGISTERED != -1L) != 0)) { | ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ src/av/sidedata/sidedata.c:5662:9: note: each undeclared identifier is reported only once for each function it appears in error: command '/usr/bin/gcc' failed with exit code 1 [end of output] note: This error originates from a subprocess, and is likely not a problem with pip. ERROR: Failed building wheel for ha-av ERROR: Could not build wheels for ha-av, which is required to install pyproject.toml-based projects



`

**Reply by Scialla** [reply]
se cerchi l'errore su google trovi soluzioni diverse, sta a te provarle sperando di trovare quella giusta.
Non avendo più una installazione in Venv diventa difficile aiutarti.
https://www.google.com/search?q=Unable+to+install+package+ha-av%3D%3D10.1.0%3A+error%3A+subprocess-exited-with-error&rlz=1C1GCEA_enIT947IT947&sxsrf=AB5stBiF4ridMnKyzlSZdpczIbjQjBNPmA%3A1688369782798&ei=dnqiZOWkMK6Jxc8PlfSRiAE&ved=0ahUKEwjlt5zJg_L_AhWuRPEDHRV6BBEQ4dUDCA8&uact=5&oq=Unable+to+install+package+ha-av%3D%3D10.1.0%3A+error%3A+subprocess-exited-with-error&gs_lcp=Cgxnd3Mtd2l6LXNlcnAQA0oECEEYAFAAWABg0gRoAHABeACAAXiIAXiSAQMwLjGYAQCgAQKgAQHAAQE&sclient=gws-wiz-serp

i primi tre link hanno tre soluzioni diverse... hai già provato?

**Reply by Ilmessaggeromascherato** [reply]
Ciao si avevo già visto quella guida. 
Stò comunque considerando il discorso Docker, e siccome mi sembra di capire che sei un esperto in materia ti chiedo: 
E' possibile migrare la configurazione dall'attuale HA a quello in DOCKER ? è sufficiente ripristinare un backup ? 
Ho installato HACS e MQTT Broker quando faccio un aggiornamento Docker devo reinstallarli ? 
Grazie per l'aiuto

**Reply by snakuzzo** [reply]
Esperto mo... non esagerare :D

Comunque sì, si fa facile...basta fare il bind della directory attuale quando vai a creare il container docker ( è tutto in guida ).
Dovrai probabilmente solo adeguare qualche "ca**atina" qua e là...ma tanto te ne accorgerai dai log di Core al primo start

**Reply by indomus** [reply]
https://indomus.it/guide/come-migrare-home-assistant-core-da-ambiente-venv-a-docker-su-raspberry-pi/

**Reply by snakuzzo** [reply]
altro consiglio (oltre a quello di capire un attimino come funziona docker) ...usa docker-compose

**Comment by Ilmessaggeromascherato** [comment]
Ciao Snakuzzo 
allora ieri sera ciò lavorato e HA si avvia su docker.... ho seguito la guida... non capisco però come far vedere la configurazione di HA esistente su docker, come importarla... 
Mi dai ancora qualche diritta ? 
Grazie

**Reply by snakuzzo** [reply]
Te l'ho data ieri. Devi fare il bind della directory di configurazione di Core... Tra le altre cose è tutto scritto al link postato ieri
